binary/proto/secret.go

```diff
@@ -220,6 +237,14 @@ func gcpsakToStruct(sakPB *spb.SecretData_GCPSAK) velesgcpsak.GCPSAK {
 	return sak
 }
 
+func anthropicAPIKeyToStruct(keyPB *spb.SecretData_AnthropicAPIKey) veles.Secret {
+	// Determine the key type based on the key content
+	if velesanthropicapikey.IsWorkspaceKey(keyPB.GetKey()) {
```

Comment:
Prefer having two separate types in the proto as well - even if it stores the same data in the end.